### ISSUE TITLE
Loading pre-trained JTNN fails.

### ISSUE THREAD

**Issue by shionhonda** [dmlc/dgl#819]
## 🐛 Bug
Loading pre-trained JTNN once after installing DGL fails.

## To Reproduce

I saw the [release note](https://www.dgl.ai/release/2019/08/28/release.html) and tried pre-trained JTNN following the example code for DGMG, but it came to the error `FileNotFoundError: [Errno 2] No such file or directory: '/home/user/.dgl/jtnn/vocab.txt'`.

```python
# We recommend running the code below with Jupyter notebooks
from IPython.display import SVG
from rdkit import Chem
from rdkit.Chem import Draw

from dgl import model_zoo

model = model_zoo.chem.load_pretrained('JTNN_ZINC')
```

## Expected behavior
It is expected to work as DGMG version.

## Environment
 - DGL Version 0.3.1:
 
## Additional context
I resolved the error by executing  `dgl/examples/pytorch/model_zoo/chem/generative_models/jtnn/train.py`.
It prepares the vocabulary file in `.dgl/jtnn/vocab.txt`

**Comment by VoVAllen** [comment]
Thanks. I'll update the script to prepare the dataset.

**Comment by mufeili** [comment]
@VoVAllen Any progress?

**Comment by mufeili** [comment]
@shionhonda Sorry for the late reply. This should now be fixed with #843 .
